gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactor.java

```diff
@@ -125,8 +114,8 @@
   public static final String COMPACTION_HIGH_PRIORITY_TOPICS = COMPACTION_PREFIX + "high.priority.topics";
   public static final String COMPACTION_NORMAL_PRIORITY_TOPICS = COMPACTION_PREFIX + "normal.priority.topics";
   public static final String COMPACTION_JOBPROPS_CREATOR_CLASS = COMPACTION_PREFIX + "jobprops.creator.class";
-  public static final String DEFAULT_COMPACTION_JOBPROPS_CREATOR_CLASS =
-      "gobblin.compaction.mapreduce.MRCompactorTimeBasedJobPropCreator";
+  public static final String DEFAULT_COMPACTION_JOBPROPS_CREATOR_CLASS = MRCompactorJobPropCreator.class.getName();
```

Comment:
Since now there is only one job prop creator class, you can remove the reflection.